tests/test_to_container.py

```diff
@@ -167,3 +167,121 @@ def test_string_interpolation_with_readonly_parent() -> None:
 def test_to_container_missing_inter_no_resolve(src: Any, expected: Any) -> None:
     res = OmegaConf.to_container(src, resolve=False)
     assert res == expected
+
```

Comment:
That's a bit too many tests.
Let's try to keep tests high value, a lot of test code becomes a maintenance burden.

I think we can go with a single test function testing:
1. with and without enum_to_str
2. A single config containing dict and list, enums as keys and enums as values.

Maybe:
```python
cfg = {
  "d": {
    Color.RED: "red",
    "red" : Color.RED,   
  },
  "l" : ["red", Color.RED]
}
```

Testing conversion of a config derived from this, with the flag true and false should give solid coverage and will be much more compact.

Reply:
Done in https://github.com/omry/omegaconf/pull/549/commits/812ce8018e466308ef80d6ad3ab7812971c86da6